runtests.jl

```diff
@@ -26,7 +26,7 @@ for exercise_type in ("concept", "practice")
         # runtests.jl includes the solution by calling `include("slug.jl")`
         # Our anonymous module doesn't have `include(s::String)` defined,
         # so we define our own.
-        Core.eval(m, :(include(s) = $(Base.include)($m, $(joinpath(exercise_path, ".meta", example_or_exemplar * ".jl")))))
+        @eval m include(s) = Base.include($m, joinpath(".meta", $example_or_exemplar * ".jl"))
```

Comment:
Does this still do what you want it to do?

I fixed the path and changed it to use `@eval` like the other defs in this file

Reply:
Looks fine to me :)